### PR TITLE
Add release notes from 0.13.x to 0.14.x

### PR DESCRIPTION
**Pull request by savitaashture**
/cc @khrm

### CODE REVIEW

File: openshift/release/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+* :book: Rewrite triggers.md for clarity and flow (#992)
+* :book: Update descriptions for `TriggerTemplates` and `TriggerBindings` (#991)
+* :book: Rewrite triggerbindings.md for clarity and flow (#988)
+* :book: Rewrite triggertemplates.md for clarity and flow (#981)
+* :book: Update examples to use both old and new syntax (#1028)
+* :book: Update eventlisteners.md (#993)
+* :book: Correct eventlistener docs to properly set service account for a trigger (#984)
+* :book: Add link to versioned docs for v0.12.0 (#982)
+* :book: fix a typo (#979)
+* :book: Fix broken link to Getting Started (#978)
```
Comment (savitaashture) [inline openshift/release/release-notes.md:172]: we can remove

File: openshift/release/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+
+## Misc
+
+* :hammer: Update triggers, community and cli reference to use main… (#987)
+* :hammer: Use v1 API for RoleBinding 🦝 (#1046)
+* :hammer: Add separate interceptors file result to release pipeline (#1044)
+* :hammer: Update publish task to include core interceptors (#1007)
+* :hammer: Update Release Pipeline and associated Task 🧑‍💻 😺 (#1000)
+* :hammer: Add links to versioned docs for v0.12.1 (#999)
```
Comment (savitaashture) [inline openshift/release/release-notes.md:154]: we can remove

File: openshift/release/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+## Features
+
+* :sparkles: Adding label selector for triggers [#970](https://github.com/tektoncd/triggers/pull/970)
+
+    Users can now configure triggers for an EventListener using labels
+
+*  :sparkles:  Added URL field to WebhookInterceptor (#1002)
+
+    Users can now configure webhook interceptors by referencing a URL
```
Comment (savitaashture) [inline openshift/release/release-notes.md:100]: how about removing this

File: openshift/release/release-notes.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    a `params` field to add parameters to pass on to the interceptor for processing
+
+*  :sparkles:  Migrate core interceptors to use InterceptorType CRD (#976)
+
+    The four bundled interceptors (CEL, GitHub, GitLab, BitBucket) are now implemented using the new ClusterInterceptor CRD
+
+*  :sparkles:   Migrate core interceptors to new format (#1029)
+
+     Any new Triggers created using the old style syntax for core interceptors is now automatically switched to the new refs/params 
+     based syntax.
```
Comment (savitaashture) [inline openshift/release/release-notes.md:118]: @khrm How about combining all these into single sentence and linking those PR fixes

/cc @Preeticp @sounix000

File: openshift/release/release-notes.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+* :book: Add the new `Interceptors` Page (#1035)
+* :book: Rewrite the `ClusterInterceptors` page for clarity and flow (#1048)
+* :book: Capitalization consistency for resourcetemplates (#1050)
+* :book: Update README.md and Release Cheat Sheet (#1051)
+* :book: Update install documentation (#1054)
+* :book: Rewrite the `EventListeners` Page (#1060)
+* :book: Delete obsolete page exposing-eventlisteners.md (#1066)
+* :book: Update link to interceptors doc (#1073)
+* :book: Add link between cluster interceptor docs and interceptor docs (#1074)
```
Comment (savitaashture) [inline openshift/release/release-notes.md:88]: we can remove this

File: openshift/release/release-notes.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+* :hammer: Resolving build test issues in PRs (#1078)
+* :hammer: Migrate from test builders to structs (#1080)
+* :hammer: Bump CEL version to latest version. (#1088)
+
+The CEL interceptor is updated to the latest version of CEL, this brings in the CEL base64 decoding functions.
+These are documented here https://github.com/google/cel-go/tree/master/ext#base64decode
+**Note:** the CEL base64 decoding function decodes to a CEL byte array, which is not a string, it uses the b'hello' syntax to represent a string of bytes "hello".
+The triggers `decodeb64` CEL function is now deprecated in favour of the upstream encoders package, you can replace `body.value.decodeb64` with `string(base64.decode(body.value))`
+
+* :hammer: Bump pipeline to 0.24.1 and knative pkg to 0.22 (#1091)
```
Comment (savitaashture) [inline openshift/release/release-notes.md:75]: we can remove this

File: openshift/release/release-notes.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+
+Skip *heavy* validation on deletion in the webhook.
+
+* :bug: Fix metrics port expose issue for triggers eventlistener (#1086)
+
+Triggers exposes metrics on `9000` port.
+
+* :bug: Add EventListener UID to sink response, mark name/namespace as deprecated (#1087)
+
+`eventListener` and `namespace` fields in EventListener response are now deprecated. Use `eventListenerUID` instead.
```
Comment (savitaashture) [inline openshift/release/release-notes.md:58]: we can remove these

File: openshift/release/release-notes.md [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+TriggerBinding produces error on providing invalid Tekton template values.
+
+* :sparkles: Adding opencensus metrics (#1061)
+
+Added Eventlistener OpenCensus metrics which captures metrics at process level.
+
+* :sparkles: Add validation check for triggers (#1072)
+
+User cannot specify both TriggerRef and (TriggerTemplate, TriggerBindings, Interceptors) as part of eventlistener spec.
+
```
Comment (savitaashture) [inline openshift/release/release-notes.md:23]: we can remove this

File: openshift/release/release-notes.md [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+Handled deprecation of `podTemplate` properly so that there won't be any issue with Upgrade which we faced as part of `0.14.0`.
+Reference Issue https://github.com/tektoncd/triggers/issues/1098
+
+# Tekton Triggers v0.14.0
+
+## Features
+* :sparkles: Provide error for invalid bindings (#1049)
+
+TriggerBinding produces error on providing invalid Tekton template values.
+
```
Comment (savitaashture) [inline openshift/release/release-notes.md:15]: this also we can remove

File: openshift/release/release-notes.md [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+* :sparkles: Add a Ready StatusCondition for EventListener (#1082)
+
+The EventListener Status now has a new Condition called Ready. Ready is set to True when the other status conditions are also true (i.e. the EventListener is ready to serve traffic). It is false if any of the other statuses are false.
+
+## Deprecation Notices
+
+* :rotating_light: Add a Ready StatusCondition for EventListener (#1082)
+
+The EventListener Status now has a new Condition called Ready. Ready is set to True when the other status conditions are also true (i.e. the EventListener is ready to serve traffic). It is false if any of the other statuses are false.
+
```
Comment (khrm) [inline openshift/release/release-notes.md:33]: Repetition.

File: openshift/release/release-notes.md [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+## Backwards incompatible changes
+
+* :rotating_light: Removed deprecated fields (#1040)
+
+As part of this release we have removed deprecated fields `ServiceType` and `PodTemplate` from the EventListener Spec.
+
+## Fixes
+
+* :bug: Skip *heavy* validation on deletion 🙃 (#1085)
+
```
Comment (khrm) [inline openshift/release/release-notes.md:49]: We can ignore this.

File: openshift/release/release-notes.md [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+* :bug: Fix metrics port expose issue for triggers eventlistener (#1086)
+
+Triggers exposes metrics on `9000` port.
+
+* :bug: Add EventListener UID to sink response, mark name/namespace as deprecated (#1087)
+
+`eventListener` and `namespace` fields in EventListener response are now deprecated. Use `eventListenerUID` instead.
+
+## Misc
+
```
Comment (khrm) [inline openshift/release/release-notes.md:61]: Whole section can be removed.

File: openshift/release/release-notes.md (hunk 4 above)
Comment (khrm) [reply openshift/release/release-notes.md:118]: Agreed.

File: openshift/release/release-notes.md (hunk 10 above)
Comment (savitaashture) [reply openshift/release/release-notes.md:22]: removed the repeated one

File: openshift/release/release-notes.md (hunk 4 above)
Comment (savitaashture) [reply openshift/release/release-notes.md:59]: @Preeticp @sounix000 could you take care of this while adding doc

**Review by sounix000 — APPROVED** [review]

**Pull request closed without merging**